configs/starfive/visionfive2/kernel_patches/0001-FIX_FOR_GCC12.patch

```diff
@@ -0,0 +1,39 @@
+diff -u -r ./arch/riscv/Makefile ../../SkiffOS/workspaces/starfive/build/linux-custom/arch/riscv/Makefile
+--- ./arch/riscv/Makefile	2023-01-11 16:29:35.000000000 +0100
++++ ../../SkiffOS/workspaces/starfive/build/linux-custom/arch/riscv/Makefile	2023-02-21 23:16:06.176199599 +0100
```

Comment:
Minor point: it doesn't really matter all that much (this works), but to generate nicely formatted patches;

 1. Git clone and checkout the Linux kernel version defined in the config.
 2. Apply your changes.
 3. Make a git commit with the changes.
 4. Format the patches with `git format-patch -N HEAD~1` (or whatever the base version was).

This formats the patches with git format and adds proper filename formatting + commit messages. And it makes it much easier to rebase the patches in future using "git am" and then "git rebase"